source/style-guide/style/product-names-version-numbers.txt

```diff
@@ -19,18 +19,18 @@ the following guidelines:
    * - Spell out and properly capitalize MongoDB product and service
        names.
      -
-       - MongoDB Atlas
-       - MongoDB Compass
+       - Atlas
+       - Compass
 
-   * - Use the full MongoDB product name on the first mention.
-       Subsequent mentions may use the product name without MongoDB.
```

Comment:
Lines 25 and 26 still hold true because they refer to the page's body, but I don't feel strongly about keeping "MongoDB" in the body text.